### ISSUE TITLE
Pytorchnn support multi-gpus

### ISSUE THREAD

**Issue by yzmyyff** [kaldi-asr/kaldi#4468]
Pytorchnn only uses one gpu. It's very inefficient when training a large amount of text.

**Comment by kkm000** [comment]
This is a Kaldi repository. Whet do you mean? K2 is here, btw: https://github.com/k2-fsa/k2.

**Comment by yzmyyff** [comment]
Please see codes in local/pytorchnn. It's another nnlm rescore backend.

**Comment by kkm000** [comment]
My best guess is that it's not implemented. A PR is welcome.

@keli78, do you have something to add?

**Comment by keli78** [comment]
@kkm000 Right, we don't have a multi-gpu implementation. I'm afraid I'm not available to add it currently. It will be great if you would like to take it.

**Comment by kkm000** [comment]
@keli78, I'm not up to the task. There is too much work undone on Kaldi that my plate is full, and I have a dayjob (although they do intersect a lot).

@yzmyyff, if you want to develop it, that would be very welcome!

**Comment by yzmyyff** [comment]
I'm sorry that I am not in this direction now. 

The conclusion of the previous attempt is that it will obviously reduce server throughput.

So keeping the existing status is enough for me.
